infrastructure/json/src/test/java/tech/pegasys/teku/infrastructure/json/types/DeserializableOneOfTypeDefinitionBuilderTest.java

```diff
@@ -22,11 +22,9 @@
 import tech.pegasys.teku.infrastructure.json.JsonUtil;
 
 public class DeserializableOneOfTypeDefinitionBuilderTest {
-  public static final DeserializableOneOfTypeDefinition<
-          OneOfTypeTestTypeDefinition.TestType, TestTypeBuilder>
+  public static final DeserializableOneOfTypeDefinition<OneOfTypeTestTypeDefinition.TestType>
       DESERIALIZABLE_ONE_OF_TYPE_DEFINITION =
-          DeserializableOneOfTypeDefinition.object(
-                  OneOfTypeTestTypeDefinition.TestType.class, TestTypeBuilder.class)
+          DeserializableOneOfTypeDefinition.object(OneOfTypeTestTypeDefinition.TestType.class)
```

Comment:
We are not using `TestTypeBuilder` anymore, we could delete it?

Reply:
Yep, actually we don't use the builder. The whole `DeserializableOneOfTypeDefinition` was added back when we refactored the block v2 API to support DENEB but was never used at the end (Tested only). I don't see the need for a builder...